internal/exec/stages/disks/luks.go

```diff
@@ -41,8 +41,10 @@ var (
 
 // https://github.com/latchset/clevis/blob/master/src/pins/tang/clevis-encrypt-tang.1.adoc#config
 type Tang struct {
-	URL        string `json:"url"`
-	Thumbprint string `json:"thp,omitempty"`
+	URL                 string `json:"url"`
+	Thumbprint          string `json:"thp,omitempty"`
+	OfflineProvisioning bool   `json:"offlineProvisioning,omitempty"`
```

Comment:
Is this a real field?  I don't see it mentioned in the docs link above, and we're not setting it.

Reply:
No its not; Yeah it should not be here.